docs/source/admin/tasks.rst

```diff
@@ -2,6 +2,10 @@
 Admin Tasks
 ===========
 
+Admin tasks describes processes for the intialization/deletion of the registry and reference dictionnary updates.
```

Comment:
Minor typo: "Admin **tasks** **describes**" should be "Admin **tasks** **describe**"

Reply:
Done